thavalon-webapp/src/components/Register.tsx

```diff
@@ -1,30 +1,86 @@
-import React, {useEffect, useState} from 'react';
+import React, {useState} from 'react';
 import ReactModal from 'react-modal';
-import { useForm } from 'react-hook-form';
-import "./modal.scss";
-import { register_user } from '../utils/account_utils';
+import { Resolver, useForm } from 'react-hook-form';
+import "./Modal.scss";
+import { AccountManager, HttpResponse } from '../utils/accountManager';
+import { Redirect } from 'react-router-dom';
 
+// Used by react modal for screen readers
 ReactModal.setAppElement("#root");

```

Comment:
Do we use `ReactModal` in other files?

Reply:
Yeah we do - login also uses it. Should I just set this once in App?

Reply:
I think so, since it's not specific to just one or the other